Sources/OpenCombineFoundation/PropertyListEncoder.swift

```diff
@@ -8,10 +8,12 @@
 import Foundation
 import OpenCombine
 
+#if swift(>=5.1)
```

Comment:
It seems this is only true with swift-corelibs-foundation. With Darwin Foundation, it works even on Swift 5.0.
```suggestion
// PropertyListEncoder and PropertyListDecoder are unavailable in swift-corelibs-foundation prior to Swift 5.1.
#if !canImport(Darwin) && swift(>=5.1)
```